cedar-policy-validator/src/extensions/ipaddr.rs

```diff
@@ -32,8 +33,11 @@ use std::str::FromStr;
 
 // PANIC SAFETY see `Note on safety` above
 #[allow(clippy::panic)]
-fn get_argument_types(fname: &str, ipaddr_ty: &Type) -> Vec<types::Type> {
-    match fname {
+fn get_argument_types(fname: &Name, ipaddr_ty: &Type) -> Vec<types::Type> {
+    if !fname.is_unqualified() {
+        panic!("unexpected decimal extension function name: {fname}")
```

Comment:
Nit 😉 I realize that this code should be unreachable.
```suggestion
        panic!("unexpected ipaddr extension function name: {fname}")
```